### ISSUE TITLE
Add training data

### ISSUE THREAD

**Issue by divyansh335500** [a-maliarov/amazoncaptcha#126]
Upload your file or paste the links...
![test captcha amazon](https://github.com/user-attachments/assets/dc683976-13e3-420d-ac6d-263bbee29c64)

please train your python package to solve this type of amazon captcha

**Comment by LeM0-Dev** [comment]
Thats not an amazon market captcha @divyansh335500

**Comment by Bobrovskii** [comment]
This is a new version of captcha from Amazon. This is the version that Amazon uses now

**Comment by divyansh335500** [comment]
Yes, this is new version of captcha Amazon gives in sign in page.

**Comment by divyansh335500** [comment]
> Thats not an amazon market captcha @divyansh335500

This is sign in captcha of Amazon

**Comment by eliafino** [comment]
Me too, "not solved"

![Image](https://github.com/user-attachments/assets/1233e377-a61d-4397-a7d5-a1f27afd0149)
